src/tree/AppResourceTreeItem.ts

```diff
@@ -39,11 +39,14 @@ export class AppResourceTreeItem extends ResolvableTreeItemBase implements Group
 
     private constructor(root: AzExtParentTreeItem, resource: AppResource) {
         super(root);
-        this.rootGroupTreeItem = root;
-        this.rootGroupConfig = <GroupNodeConfiguration><unknown>root;
+
+        if (root) {
```

Comment:
I actually think these changes aren't needed anymore since this class shouldn't be used anymore. I will double check, but I think I'll revert these changes and then delete all these old classes in a follow up PR.